client/blocks/eligibility-warnings/style.scss

```diff
@@ -11,7 +11,7 @@
 		padding: 0;
 	}
 
-	> .card {
+	> .card.card {
```

Comment:
Intentional? I don't believe duplicate classnames are possible, so this is functionally the same as `.card`

Reply:
Yes that's my usual strategy for bumping up specificity. That way we can be sure the extra classname is there for specificity purposes only, as opposed to scoping, so if we get to a point where we can make it less specific, it's easy to remove the extra classname knowing it won't change the scope of the selector. I should probably add a comment in there to make that clear 😅

Reply:
TIL